packages/client/components/GitLabFieldMenu.tsx

```diff
@@ -1,13 +1,72 @@
+import styled from '@emotion/styled'
+import {Edit} from '@mui/icons-material'
 import graphql from 'babel-plugin-relay/macro'
 import {useFragment} from 'react-relay'
 import {GitLabFieldMenu_stage$key} from '../__generated__/GitLabFieldMenu_stage.graphql'
 import useAtmosphere from '../hooks/useAtmosphere'
 import {MenuProps} from '../hooks/useMenu'
+import useModal from '../hooks/useModal'
 import UpdateGitLabDimensionFieldMutation from '../mutations/UpdateGitLabDimensionFieldMutation'
+import textOverflow from '../styles/helpers/textOverflow'
+import {PALETTE} from '../styles/paletteV3'
+import {FONT_FAMILY} from '../styles/typographyV2'
 import {SprintPokerDefaults} from '../types/constEnums'
+import EditVotingLabelTemplateModal from './EditVotingLabelTemplateModal'
+import FlatButton from './FlatButton'
 import Menu from './Menu'
 import MenuItem from './MenuItem'
 
+const LabelOptionRoot = styled('div')({
+  display: 'flex',
+  justifyContent: 'space-between',
+  minWidth: '300px'
+})
+
+const LabelOptionBlock = styled('div')({
+  display: 'block',
+  flexDirection: 'column',
+  maxWidth: '200px',
+  paddingTop: 12,
+  paddingLeft: 16,
+  paddingBottom: 12,
+  flexGrow: 1
+})
+
+const LabelOptionName = styled('div')({
+  color: PALETTE.SLATE_700,
+  display: 'flex',
+  fontFamily: FONT_FAMILY.SANS_SERIF,
+  lineHeight: '24px'
+})
+
+const LabelOptionSub = styled('div')({
+  ...textOverflow,
+  color: PALETTE.SLATE_600,
+  fontFamily: FONT_FAMILY.SANS_SERIF,
+  fontSize: 12,
+  lineHeight: '16px'
+})
+
+const EditButtonGroup = styled('div')({
+  paddingLeft: '8px',
+  paddingRight: '8px',
+  marginTop: 'auto',
+  marginBottom: 'auto'
+})
+const Button = styled(FlatButton)({
+  alignItems: 'center',
+  color: PALETTE.SLATE_600,
+  height: 32,
+  justifyContent: 'center',
+  padding: 0,
+  width: 32
+})
+
+const ActionButton = styled(Edit)({
+  height: 18,
+  width: 18
+})
```

Comment:
I don't like the styled component stuff here, but I just copied it from the GitHubFieldMenu 1:1